tests/api/search/search_entities.test.js

```diff
@@ -219,6 +219,23 @@ describe('search:entities', () => {
         foundIds.should.not.containEql(humanB._id)
       })
     })
+
+    describe('offset', () => {
+      it('should offset results', async () => {
+        const types = 'works'
+        const input = 'memoire'
```

Comment:
this would mean some entities must have `memoire` as a label indexed in ES somewhere, which does not make this test isolated properly, i would be in favor of creating new entities for this test, or ensure entities are indexed somehow.

Reply:
done in e8825f03